sbin/tpm2-attest

```diff
@@ -319,7 +320,7 @@ attest()
 		"$SERVER" \
 	|| die "attestation failed"
 
-	unseal < "$TMP/cipher.tgz" \
+	unseal unseal < "$TMP/cipher.tgz" \
```

Comment:
Doesn't this extra `unseal` trigger `No arguments expected` in `unseal()`?

Reply:
Yes, it snuck in by accident and I've removed it.

Reply:
Fixed already.